src/types/FilterBy.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterBy {
+  all = 'all',
+  completed = 'completed',
+  active = 'active',
```

Comment:
```suggestion
  All = 'all',
  Completed = 'completed',
  Active = 'active',
```
Values are better to start writing in capital letters

Reply:
agree